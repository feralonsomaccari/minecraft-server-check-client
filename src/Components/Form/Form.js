import React from "react";
import css from "./Form.module.css";

const Form = () => {
  return (
    <div className={css.Form}>
      <label htmlFor="ip">Ip</label>
      <input className={css.Input} id="ip" type="text" />
      <label htmlFor="port">Port</label>
      <input className={css.InputPort} id="port" type="text" />
    </div>
  );
};

export default Form;
