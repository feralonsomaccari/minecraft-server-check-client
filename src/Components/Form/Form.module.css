.Form {
  display: flex;
  justify-content: center;
  align-items: center;
  padding: 1rem;
}
.Form label {
  font-family: "minecraft";
  color: #fff;
  margin: 0 1rem;
  text-shadow: 1px 1px black;
}

.Input {
  font-family: "minecraft";
  margin: 0 1rem;
  width: 15rem;
  padding: 0.3rem;
}
.InputPort {
  font-family: "minecraft";
  padding: 0.3rem;
  width: 5rem;
}
