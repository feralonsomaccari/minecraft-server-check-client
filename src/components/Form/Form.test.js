import React from "react";
import { shallow } from "enzyme";
import Form from "./Form"


describe("<Form/>", () => {

    const wrapper = shallow(<Form></Form>);

    it("Renders correctly", () => {
        expect(wrapper).toMatchSnapshot();
    })

    it("The ip field should have the right text", () => {
        const ipText = wrapper.find("label#form-ip").text();
        expect(ipText).toBe("Ip")
    })

    it("the port field have the right text", () => {
        const portText = wrapper.find("label#form-port").text();
        expect(portText).toBe("Port")
    })

})