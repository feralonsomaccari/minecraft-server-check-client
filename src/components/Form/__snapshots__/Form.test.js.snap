// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Form/> Renders correctly 1`] = `ShallowWrapper {}`;
